src/renderer/shared/lib/utils/balance.ts

```diff
@@ -86,7 +86,7 @@ export const formatBalance = (balance = '0', precision = 0): FormattedBalance =>
   };
 };
 
-export const totalAmount = (balance?: Balance): string => {
+export const totalAmount = (balance?: Balance | TokenBalance): string => {
```

Comment:
I suppose something like this should work
```suggestion
export const totalAmount = <T extends TokenBalance>(balance?: T): string => {
```